Sources/AppleAPI/Client.swift

```diff
@@ -56,6 +61,90 @@ public class Client {
             }
     }
     
+    public func srpLogin(accountName: String, password: String) -> Promise<Void> {
+        var serviceKey: String!
+        let client = SRPClient(configuration: SRPConfiguration<SHA256>(.N2048))
+        let clientKeys = client.generateKeys()
+        let a = clientKeys.public
+        
+        return firstly { () -> Promise<(data: Data, response: URLResponse)> in
```

Comment:
Non-blocking, but it would be good to have a small comment around each of the steps here since it seems like an involved handshake process